cmd/regex_format.go

```diff
@@ -326,16 +326,48 @@ func findUpperCaseOnIgnoreCaseFlag(lines []string, iFlag bool) (bool, string) {
 	// check if the file contains uppercase letters
 	if iFlag {
 		for number, line := range lines {
-			// ignore line if it starts with ##!
-			if strings.HasPrefix(line, "##!") {
+			// ignore line if it starts with the following regex: \s+##!
+			if regex.CommentRegex.MatchString(line) {
 				continue
 			}
-			found := strings.IndexAny(line, "ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+			found := findUppercaseNonEscaped(line)
 			if found > -1 {
```

Comment:
Check the boolean here.
```suggestion
			if found {
```